test/services/xccdf_report_parser_test.rb

```diff
@@ -55,22 +55,6 @@ def package_name
       end
     end
 
-    should 'not save missing benchmarks for RHEL8' do
```

Comment:
@skateman Should this test also be removed?

Reply:
Yes, as the saving code is deleted, so we cannot really test its opposite.

Reply:
Confirmed, that the saving logic is intended to be removed.